client/scss/core.scss

```diff
@@ -198,4 +198,4 @@ These are classes that provide overrides.
 @import 'layouts/chooser-duplicate-upload';
 @import 'layouts/focal-point-chooser';
 @import 'layouts/modeladmin';
-@import 'layouts/redirects';
+@import 'layouts/redirects';
```

Comment:
Can this unrelated change be removed - it seems like your editor may be set to remove lines at the end of the file. 

Here's an article on how to fix this in VSCode. 
https://blog.boot.dev/clean-code/line-breaks-vs-code-lf-vs-crlf/

Reply:
Okay, I will work on it.